client/src/db_interface.py

```diff
@@ -37,7 +37,7 @@ def init_db(self):
         self._logger.info("Initializing database at %s", self.conn_str)
         conn = apsw.Connection(self.conn_str)
         cur = conn.cursor()
-        cur.execute("create table if not exists ax_frame (time text, data blob);")
+        cur.execute("create table if not exists ax_frame (time text, data blob, needsRelay bit);")
```

Comment:
I feel like this "create table if not exists ax_frame" text could be changed into something more readable.
Secondly, we use snake_case in Python codem "needsRelay" isn't snake_case!

Reply:
Disregard the first section of this comment.